install/helm/agones/templates/service-monitor.yaml

```
@@ -35,6 +35,7 @@ spec:
       path: /metrics
       interval: {{ .Values.agones.metrics.serviceMonitor.interval }}
       relabelings:
+{{- if and (.Values.agones.metrics.prometheusServiceDiscovery) (.Values.agones.metrics.prometheusEnabled) }}
```

```suggestion
{{- if (.Values.agones.metrics.prometheusServiceDiscovery) }}
```
prometheusEnabled is now always true